strings/group_anagram.py

```diff
@@ -9,7 +9,8 @@ def find_anagrams(_list: list) -> list[list[str]]:
     char_map = {}
     anagrams = []
     for each in _list:
-        char_map[each] = {"is_visited": False}
+        char_map[each] = {}
+        char_map[each]["is_visited"] = False
```

Comment:
Why is this an improvement?

Reply:
If char_map[each] is responsible for both tacking the visitors and the count, if it is referenced before assignment, there may be a cause of runtime error